docs/Developers/Writing_Good_Commits.rst

```diff
@@ -26,6 +26,9 @@ Example of a good commit:
 - `Fixes https://github.com/coala/coala/issues/5861`: Mention the URL
    of the issue it closes or fixes.
 
+Use 'Fixes' when you fixed a bug and 'Closes' when your work is
```

Comment:
@userzimmermann Actually the original one has grammatical errors, it should either be: 
`Use 'Fixes' when you fix a bug and 'Closes' when your work is ... `
or
`Use 'Fixes' when you have fixed a bug and 'Closes' when your work is ... `
Also, the changes I suggested are not my own(would have got errors myself :stuck_out_tongue: , they are the messages from the @gitmate-bot . With this we will have uniformity in messages on the tutorial and the gitmate bot. See here the point from the bot message:
> - I have used the Fixes keyword if the commit fixes a real bug and the Closes keyword if it adds a feature/enhancement. See more here.